mlos_bench/mlos_bench/config/services/remote/azure/scripts/setup-rg/README.md

```diff
@@ -29,23 +29,42 @@ The *control plane RG* is a container for the *persistent* resources of MLOS (re
 
     ```shell
     # With Powershell, recommended to use Powershell 7
+    # If provisioning results DB include '-resultsDbArmsParamsFile', otherwise omit
+
+    # If setting up with a Service Principal
     ./setup-rg.ps1 `
         -controlPlaneArmParamsFile $controlPlaneArmParamsFile `
-        -resultsDbArmParamsFile $resultsDbArmParamsFile  # If provisioning results DB, otherwise omit `
-        -servicePrincipalName $servicePrincipalName `
         -resourceGroupName $resourceGroupName `
+        -resultsDbArmParamsFile $resultsDbArmParamsFile `
+        -servicePrincipalName $servicePrincipalName `
         -certName $certName
+
+    # If setting up with a Managed Identity
```

Comment:
Can you give more context in the README about this part?
When should one use a Managed Identity? How does one set it up, etc.